app/controllers/ProdMonitoring.scala

```diff
@@ -0,0 +1,32 @@
+package controllers
+
+import actions.CustomActionBuilders
+
+import com.gu.tip.Tip
+import play.api.libs.circe.Circe
+import play.api.mvc._
+
+import scala.concurrent.{ExecutionContext, Future}
+
+class ProdMonitoring(
+    actionBuilders: CustomActionBuilders,
+    components: ControllerComponents,
+    tipMonitoring: Tip
+)(implicit val ec: ExecutionContext) extends AbstractController(components) with Circe {
+
+  import actionBuilders._
```

Comment:
personally opinion is I don't favour this kind of local import, just because it's harder to review where things come from.